### DISCUSSION TITLE
how to deal with static proxy by apisix?

### DISCUSSION THREAD

**Discussion by chenyongzhi001** [apache/apisix#7928] [Q&A]
![YX7XCNOO5 VVNYXW{OOS1DG](https://user-images.githubusercontent.com/35951080/190546823-5514d821-cb27-48fb-a40c-4c5a32aab769.png)
as picture , how to deal with static proxy by apisix?

**Comment by tokers** [comment]
Hi, you can add these configurations via the APISIX nginx configuration snippet feature. See https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L160.

**Reply by chenyongzhi001** [reply]
> Hi, you can add these configurations via the APISIX nginx configuration snippet feature. See https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L160.

sorry, i just want to proxy_pass my staticfiles，not set nginx configuration.

**Reply by tzssangglass** [reply]
another way is: https://github.com/apache/apisix/issues/4672
